client/src/components/NavMenu/NavMenu.css

```diff
@@ -0,0 +1,89 @@
+.nav-container {
+        background-color: #FAE5DC;
+        display: flex;
+        flex-flow: column;
+        justify-content: flex-start;
+        align-content: space-around;
+        align-items: center;
```

Comment:
Looks like your nav menu stuff is in this PR too. Can you remove those changes from here too?

Reply:
@ZakkMan I removed the files from the PR.